agent/util-scripts/pbench-metadata-log

```diff
@@ -237,4 +237,6 @@ case $1 in
         exit 2
 esac
 
+sed -i 's/%/%%/g' $dir/metadata.log
+
```

Comment:
That is not going to work: the `metadata_log_end` function uses `pbench-add-metalog-option` which already escapes % signs. This is going to escape them *twice*.

Reply:
Should I add that only in metadata-log-beg option? Or I should I keep it as it was?

Reply:
It is working fine with 'metadata-log-beg' option.

Reply:
I think the best thing to do is to modify the `metadata_log_start` routine: produce a  file  the way it is done currently, but store it in a *temporary* file; then at the very end, pass it through `sed` and store it in the final file. BTW, it's better to create it in a temp file *first*, rather than storing it in the final file and doing `sed -i` on that: that way the file is produced atomically, so there is less chance of somebody looking at it in its intermediate state (and getting interpolation errors).

Reply:
Seems like the recommended approach was taken.